clients/drcachesim/tracer/raw2trace.h

```diff
@@ -70,6 +71,108 @@ struct module_t {
     bool is_external; // If true, the data is embedded in drmodtrack custom fields.
 };
 
+/**
+ * module_mapper_t maps and unloads application modules.
+ */
+class module_mapper_t final {
+public:
+    /**
+     * Parses and iterates over the list of modules.  This is provided to give the user a
+     * method for iterating modules in the presence of the custom field used by drmemtrace
+     * that prevents direct use of drmodtrack_offline_read().
+     * On success, calls the \p process_cb function for every module in the list, and
```

Comment:
I am confused: you've got doxygen comments in module_mapper_t, indicating it's public and meant for the user to use: yet raw2trace_t's handle_custom_data remains and is what is used by the tests.  Plus, this CL does not change any tests to use module_mapper_t directly, making it seem like module_mapper_t is really supposed to be a private helper to raw2trace?  Which is it?  It should go one way or the other and avoid confusion.

If module_mapper_t is supposed to be public and this is an interface change, please update the tests, potentially remove some of the raw2trace interfaces to avoid duplication, and update the changelog in api/docs/release.dox.

If module_mapper_t is private, please change all these to non-doxygen comments, and move the user-facing docs back to raw2trace.

Reply:
Oh - I was on the assumption we needed to keep around the existing APIs for back-compat. Is it OK to delete them (e.g. raw2trace_t::find_mapped_trace_address, do_module_parsing, etc)?